pkg/oc/oc.go

```diff
@@ -117,3 +120,65 @@ func CreateSecretsForTektonResults(){
 	//creating secret with generated certificate
 	cmd.MustSucceed("oc", "create", "secret", "tls", "-n", "openshift-pipelines", "tekton-results-tls", "--cert=cert.pem", "--key=key.pem")
 }
+
+func EnsureResutsReady(){
+	cmd.MustSucceed("oc", "wait", "--for=condition=Ready", "tektoninstallerset", "-l", "operator.tekton.dev/type=result")
+}
+
+func CreateResultsRoute(){
+	cmd.Run("oc", "create", "route", "-n", "openshift-pipelines", "passthrough", "tekton-results-api-service", "--service=tekton-results-api-service", "--port=8080")
+}
+
+func ConfigureResultsApi() string{
```

Comment:
It runs `oc get`, it doesn't configure anything so the name of function is a bit misleading